hearpreprocess/pipeline.py

```diff
@@ -26,6 +28,9 @@
 TRAIN_PERCENTAGE = 100 - VALIDATION_PERCENTAGE - TEST_PERCENTAGE
 
 # We want no more than 5 hours of audio per task.
+# This can be overriden in the task config.
+# e.g. speech_commands test set.
+# If None, no limit is used.
```

Comment:
Maybe include an example on how to do that, or least what key to use in task config?